test/parallel/test-readline-interface.js

```diff
@@ -305,6 +305,67 @@ function isWarned(emitter) {
     return false;
   });
 
+  // duplicate lines are removed from history when `options.deDupeHistory`
+  // is `true`
+  fi = new FakeInput();
+  rli = new readline.Interface({
+    input: fi,
+    output: fi,
+    terminal: true,
+    deDupeHistory: true
+  });
+  expectedLines = ['foo', 'bar', 'baz', 'bar', 'bat', 'bat'];
+  callCount = 0;
+  rli.on('line', function(line) {
+    assert.equal(line, expectedLines[callCount]);
```

Comment:
It would probably be preferred if these assertions could all be their `strict` variants, e.g. `assert.strictEqual()` or `asset.notStrictEqual()`.